### PR TITLE
Customization vignette

### LINKED ISSUE #94: Document all customizations required to repurpose the app
Vignette (or something) that describes the specific customizations one would need to reuse the app in a different consortium (until it is more configurable).

### PR DESCRIPTION
**Pull request by karawoo**
Closes #94

### CODE REVIEW

**Comment by kelshmo** [comment]
I'm throwing a pandoc error when I try to build the site `Error: callr subprocess failed: callr subprocess failed: pandoc document conversion failed with error 99`.  I recall you encountering some errors with image resources previously. Is this workaround how you resolved the error - [ref to `pkgdown` issue](https://github.com/r-lib/pkgdown/issues/1163)?

**Comment by karawoo** [comment]
@kelshmo there was a bug in rmarkdown that caused this issue for me; I was able to solve it by installing the development version of rmarkdown (which is now captured in renv.lock). Can you confirm which version of rmarkdown you are using, and whether your packages are up-to-date with the renv lockfile?

**Comment by kelshmo** [comment]
That worked!

File: vignettes/customizing-dccvalidator.Rmd [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+1. Deploy the application as described in the 
+   [Deploying dccvalidator](https://sage-bionetworks.github.io/dccvalidator/articles/deploying-dccvalidator.html) 
+   vignette.
+
+## Configuration options
+
+* `parent`: The Synapse project or folder where files will be stored
+* `study_table`: Synapse ID of a table that lists all of the existing studies in
+  the consortium
+* `annotations_table`: Synapse ID of a table that lists allowable annotation
```
Comment (kelshmo) [inline vignettes/customizing-dccvalidator.Rmd]: There is a required schema for this annotations table in order for the app not to complain. e.g. required keys, values for columnType. I suggest documenting that here.

Comment (kelshmo) [reply vignettes/customizing-dccvalidator.Rmd]: I believe this also applies to the study table.

File: vignettes/customizing-dccvalidator.Rmd [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  keys and values for the consortium
+* `annotations_link`: URL to a list or description of allowable annotation
+  values (so users can read more)
+* `templates_link`: URL to the location of metadata templates
```
Comment (kelshmo) [inline vignettes/customizing-dccvalidator.Rmd]: I recommend adding some details about the required template format as well.

File: vignettes/customizing-dccvalidator.Rmd [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+* `study_link_animal`: URL to an example study description for an animal model
+  study
+* `study_link_human`: URL to an example study description for a human study
+* `teams`: The team(s) a user must be a member of in order to use the app. If
+  the user is not in any of the teams, they will see a message telling them they
+  must be added to one of the teams.
+* `templates` (including `manifest_template`, `individual_templates`,
+  `biospecimen_templates`, and `assay_templates`): Synapse IDs of templates to
```
Comment (kelshmo) [inline vignettes/customizing-dccvalidator.Rmd]: I recommend adding details of the required template structure as well.

**Comment by karawoo** [comment]
Great suggestions, will do, thanks @kelshmo!

**Comment by karawoo** [comment]
ugh I messed up a rebase...stand by...

**Comment by karawoo** [comment]
ok @kelshmo I think this is ready for another look

**Pull request merged**